frontend/src/components/pages/cabins/Documents/FAQ.tsx

```diff
@@ -22,9 +22,24 @@ const faqs = [
   {
     question: "Hvor lang er avbestillingsfristen?",
     answer: (
-      <Typography>
-        Avbestillingsfristen er 2 uker før ankomst. Hvis dere avbestiller etter dette, ilegges det en gebyr.
-      </Typography>
+      <Box>
+        <Typography>Følgende regler gjelder for avbestilling av en booking:</Typography>
+        <ul>
+          <li>
+            <Typography>Avbestilling innen to uker før avreise er gebyrfritt.</Typography>
+          </li>
+          <li>
+            <Typography>
+              Avbestilling innen en uke i forkant av av avreise medfører et gebyr på 30% av leien.
+            </Typography>
+          </li>
+          <li>
+            <Typography>
+              Avbestilling innen en uke i forkant av av avreise medfører et gebyr på 50% av leien.
```

Comment:
I guess this is Hyttestyret's domain, but would it be more accurate to refer to the deadlines for cancellations as prior to "bestilt dato", or something similar? If you never depart, then these don't really apply, do they?

```suggestion
            <Typography>Avbestilling innen to uker før avreise er gebyrfritt.</Typography>
          </li>
          <li>
            <Typography>
              Avbestilling innen en uke i forkant av av avreise medfører et gebyr på 30% av leien.
            </Typography>
          </li>
          <li>
            <Typography>
              Avbestilling innen en uke i forkant av av avreise medfører et gebyr på 50% av leien.
```